GVFS/GVFS.Common/InstallerPreRunChecker.cs

```diff
@@ -245,19 +248,28 @@ private bool IsGVFSUpgradeAllowed(out string error)
 
             if (!this.IsElevated())
             {
-                error = "The installer needs to be run from an elevated command prompt.\nPlease open an elevated (administrator) command prompt and run gvfs upgrade again.";
+                error = string.Join(
+                    Environment.NewLine,
+                    "The installer needs to be run from an elevated command prompt.",
+                    "Run `gvfs upgrade --confirm` again from an elevated command prompt.");
                 return false;
             }
 
             if (!this.IsProjFSInboxed())
             {
-                error = "Unsupported ProjFS configuration.\nCheck your team's documentation for how to upgrade.";
+                error = string.Join(
+                    Environment.NewLine,
+                    "Unsupported ProjFS configuration.",

```

Comment:
We are supporting this configuration of ProjFS, it just doesn't allow for auto-upgrade.  

Should we say something like "ProjFS configuration does not support \`gvfs upgrade\`"?